guides/common/modules/proc_exporting-a-content-view-version-incrementally-in-a-syncable-format.adoc

```diff
@@ -7,7 +7,7 @@ The size of the exported content view versions depends on the number of products
 endif::[]
 Content view versions that have multiple {RHEL} trees can occupy several gigabytes of space on {ProjectServer}.
 
-In such cases, you can use *Incremental Export* to export only pieces of content that changed since the previous export.
+In such cases, you can create an incremental export which contains only pieces of content that changed since the previous export.
```

Comment:
```suggestion
In such cases, you can create an incremental export which contains only pieces of content that have changed since the last export.
```

Reply:
Applied (twice)